Libplanet.Tests/TestUtils.cs

```diff
@@ -171,11 +171,11 @@ public static byte[] GetRandomBytes(int size)
 
         public static Block<T> MineGenesis<T>(
             Address? miner = null,
-            IEnumerable<Transaction<T>> transactions = null
+            IEnumerable<Transaction<T>> transactions = null,
+            DateTimeOffset? timestamp = null
         )
             where T : IAction, new()
         {
-            var timestamp = new DateTimeOffset(2018, 11, 29, 0, 0, 0, TimeSpan.Zero);
```

Comment:
How about
```
timestamp ??= new DateTimeOffset(2018, 11, 29, 0, 0, 0, TimeSpan.Zero);
```
like this?

Reply:
@riemannulus We don't use C# 8 yet in the test projects… 😢

Reply:
Ah, I forgot about that. 😭